### DISCUSSION TITLE
Open previewer / designer consumes CPU and GPU

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#10764] [Q&A] [answered]
I have tried to find related discussions as I thought I had seen that in the past but I was not able to find it...

If I have the previewer / designer open in Visual Studio 2022 (17.5.3) using Avalonia 11.0.0-preview6, VS consumes ~10% CPU and 25% GPU on my machine. If I close the last previewer, VS is back to "normal" (both close to 0%). This came around somewhere after preview4 and preview6.

Installed Avalonia extension: 0.10.18.2
OS: Windows 10

I believe that will be fixed with a new 11 VS extension. Is there a current workaround for this?

**Comment by jp2masa** [comment, marked as answer]
#10203
